### PR TITLE
Feature/improve docs

### PR DESCRIPTION
**Pull request by carlosmondra**
Improve docs in openreview.py (Client, Group, Tag, Invitation, and Profile classes)
Improve docs in tools.py

### CODE REVIEW

File: openreview/openreview.py [hunk 1]
```diff
@@ -20,17 +21,17 @@ class OpenReviewException(Exception):
     pass
 
 class Client(object):
-
+    """
+    :param baseurl: URL to the host, example: https://openreview.net (should be replaced by 'host' name)
+    :type baseurl: str, optional
```
Comment (pmandler) [inline openreview/openreview.py]: defaults to environmental var OPENREVIEW_BASEURL

File: openreview/openreview.py [hunk 2]
```diff
@@ -20,17 +21,17 @@ class OpenReviewException(Exception):
...
+    :param username: OpenReview username
+    :type username: str, optional
+    :param password: OpenReview password
+    :type password: str, optional
+    :param token: Session token
+    :type token: str, optional
```
Comment (pmandler) [inline openreview/openreview.py]: What is a token in this case?  What does it do?

File: openreview/openreview.py (hunk 1 above)
Comment (carlosmondra) [reply openreview/openreview.py]: Ok, I can add this

File: openreview/openreview.py (hunk 2 above)
Comment (carlosmondra) [reply openreview/openreview.py]: I believe this is the Authentication token. I can be more specific and add this

**Comment by carlosmondra** [comment]
So I added the Deprecated library in the setup.py file like so:
install_requires=[
          'pycryptodome',
          'requests>=2.18.4',
          'future',
          'nbsphinx',
          'tqdm',
          "Deprecated"
      ]
Just want to make sure that this is right

**Pull request merged**